src/pages/LandingPages/MGCovid19/CovidLandingForm.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+	<form @submit.prevent.stop="submit" novalidate>
+		<div class="row">
+			<fieldset class="small-12 columns input-wrapper recurring-amounts">
+				<legend>Choose an amount to contribute</legend>
+				<!-- TODO: Create full width toggle to replace this -->
+				<kv-toggle
+					id="kv-toggle-deposit"
+					v-model="isRecurring"
+				>
+					<label v-if="isRecurring" for="kv-toggle-deposit">Contribute monthly</label>
+					<label v-if="!isRecurring" for="kv-toggle-deposit">One-time contribution</label>
+				</kv-toggle>
+				<transition name="kvfastfade">
+					<kv-pill-toggle
+						v-show="isRecurring"
+						id="amount-selector"
+						:options="amountOptions"
+						:selected="amountSelected"
+						@pill-toggled="selectAmount"
+						:split-pills="true"
+					/>
+				</transition>
+				<transition name="kvfastfade">
+					<div class="custom-amount-holder" v-show="isRecurring && customAmountActive">
+						<kv-currency-input
+							id="custom-amount-input"
+							class="input-element"
+							v-model="customAmount"
+							:aria-disabled="!customAmountActive"
+							:disabled="!customAmountActive"
+						/>
+					</div>
+				</transition>
+				<transition name="kvfastfade">
+					<div class="custom-amount-holder" v-show="!isRecurring">
+						<kv-currency-input
+							id="onetime-amount-input"
+							class="input-element"
+							v-model="onetimeAmount"
+						/>
+					</div>
+				</transition>
+			</fieldset>
+			<!-- TODO: Add inline validations once design is finalized -->
+			<!-- <fieldset class="large-4 medium-5 small-12 columns input-wrapper">
+				<label class="show-for-sr" :class="{ 'error': $v.$invalid }" :for="'amount' + componentKey">
+					Amount
+				</label>
+				<kv-currency-input :id="'amount' + componentKey" :value="amount" @input="updateAmount" />
+				<ul class="validation-errors" v-if="$v.$invalid">
+					<li v-if="!$v.amount.required">
+						Field is required
+					</li>
+					<li v-if="!$v.amount.minValue || !$v.amount.maxValue">
```

Comment:
This validation would be an easy add

Reply:
I'm about to rebuild this amount selector interface and will add it into the new version